### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by blocknomad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,38 @@
...
-4. OneGraph (https://onegraph.com) or Prisma
+1. ReasonReact with Hooks (https://reasonml.github.io/reason-react/);
+2. BuckleScript Belt (https://bucklescript.github.io/bucklescript/api/Belt.html);
+3. reason-apollo-hooks (https://github.com/Astrocoders/reason-apollo-hooks);
+4. OneGraph (https://onegraph.com) or Prisma.
 
 ## Plus
 
-Search all tweets by tweet (text)
+Search all tweets by tweet (text).
```
Comment (blocknomad) [inline README.md]: I can't understand what it is asking for

File: README.md [hunk 2]
```diff
@@ -1,42 +1,38 @@
...
 
-Hi, thanks for trying out our challenge!
-Hopefully you'll learn a lot, create an awesome project and join Astrocoders.
+Hi, thank you for taking on our challenge! We hope that you'll learn a lot, create an awesome project and join the Astrocoders team.
 
-We want to evaluate your knowledge you already have for React. And the ability to learn new techs.
-
-We want a Gmail interface that loads the last 40 tweets with the hashtag #mars.
-And search these on the searchbox.
+In this test, we will evaluate the knowledge you already have about ReactJS and your ability to learn new technologies. In order to complete it, you must develop a Gmail interface that loads the last forty tweets containing the **#mars** hashtag and enables the user to filter them using the searchbox.
```
Comment (guilhermedecampo) [inline README.md]:
```suggestion
In this test, we will evaluate the knowledge you already have about ReactJS and your ability to learn new technologies. In order to complete it, you must develop a Gmail interface that loads the last 40 (forty) tweets containing the **#mars** hashtag and enables the user to filter them using the search box.
```

File: README.md (hunk 1 above)
Comment (guilhermedecampo) [reply README.md]:
```suggestion
Open a tweet to see replies.
```

**Review by guilhermedecampo — APPROVED** [review]
Thanks for your review man! 🚀 	

Added some small changes more too

**Review by fakenickels — APPROVED** [review]

**Review by JeffersonCarvalh0 — APPROVED** [review]
Nice!

**Pull request merged**